### ISSUE TITLE
omf theme completions show available but not installed themes

### ISSUE THREAD

**Issue by derekstavis** [oh-my-fish/oh-my-fish#252]

**Comment by oranja** [comment]
Based on @decors suggestion in #251 and current [code], I suggest:
```fish
complete -c omf -f -n "__fish_seen_subcommand_from t theme" -a (omf.packages.list --installed --theme)
```

I really don't know when I will get the chance to submit this, so anyone is welcome to suggest this (or something even better) in a PR.

[code]: https://github.com/oh-my-fish/oh-my-fish/blob/master/pkg/omf/completions/omf.fish

**Comment by coderstephen** [comment]
Fixed by #263.
